### DISCUSSION TITLE
Episode 10: A Talk with Amina Adewusi

### DISCUSSION THREAD

**Discussion by ckipp01** [ckipp01/tooling-talks#25] [Announcements]
Very excited to announce my next Tooling Talk with Amina! We'll chat all about newcomers getting started in Scala and tooling. How can we make it a more welcoming environment for newcomers and those underrepresented in the Scala tooling ecosystem?

![Image of Amina to advertise episode 10](https://user-images.githubusercontent.com/13974112/157028593-3ff38ac0-f31c-4a36-88f0-33a2377f03ab.jpg)

### Find Amina Online

- [@a_adewusi](https://twitter.com/a_adewusi) on Twitter
- @Nirvikalpa108 on GitHub
- [https://www.aminaadewusi.com/](https://www.aminaadewusi.com/)

**Comment by mtk** [comment]
for ms. adewusi: what do you think of scala-cli?  it is being considered as a replacement for scala/scalac.  do you use coursier?  would you be ok if coursier became the preferred method for distributing scala and its tools, including scala-cli?  with certainty, you've used scala in a programming-in-the-large environment.  but have you used it for scripting/small tasks?

**Comment by mtk** [comment]
and while it probably doesn't need to be asked since it seems to pop up in every talk, it is fascinating to hear about the tools you prefer and why.  e.g. idea?  emacs/vi?  vscode?  e.g. in the last talk, gabriel volpe explained that he used very simple tools which gave up a good deal of functionality but delivered a very simple and stable development environment.

**Reply by mtk** [reply]
sorry, two more questions.  it's the pot of coffee in me from breakfast :-).  1) scala 3 experiences/views?  i find that i love it and don't want to go back to scala 2.  i wasn't a big fan of indentation-based scoping but now i value its simplicity and readability.  2) and maybe this is more a question for chris - i was surprised to hear about an example where more than one bloop instance was started during your talk w/gabriel.  i'd like to hear more about which tools use which other tools and when.  and which versions of tools (bloop or sbt BSP server) you prefer/recommend.
